Pipfile

```diff
@@ -0,0 +1,53 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+python-slugify = "==8.0.4"
+pillow = "==10.3.0"
+argon2-cffi = "==23.1.0"
+redis = "==5.0.4"
+hiredis = "==2.3.2"
+django = "==4.2.13"
+django-environ = "==0.11.2"
+django-model-utils = "==4.5.1"
+django-allauth = {extras = ["mfa"], version = "==0.63.2"}
+django-crispy-forms = "==2.1"
+crispy-bootstrap5 = "==2024.2"
+django-redis = "==5.4.0"
+djangorestframework = "==3.15.1"
+django-cors-headers = "==4.3.1"
+drf-spectacular = "==0.27.2"
+gunicorn = "==22.0.0"
+psycopg = {extras = ["c"], version = "==3.1.19"}
+collectfast = "==2.2.0"
+django-storages = {extras = ["s3"], version = "==1.14.3"}
+django-anymail = {extras = ["sendgrid"], version = "==10.3"}
+
+[dev-packages]
+werkzeug = {extras = ["watchdog"], version = "==3.0.3"}
+ipdb = "==0.13.13"
+psycopg = {extras = ["binary"], version = "==3.1.19"}
+mypy = "==1.10.0"
+django-stubs = {extras = ["compatible-mypy"], version = "==5.0.2"}
+pytest = "==8.2.1"
+pytest-sugar = "==1.0.0"
+djangorestframework-stubs = {extras = ["compatible-mypy"], version = "==3.15.0"}
+sphinx = "==7.3.7"
+sphinx-autobuild = "==2024.4.16"
+ruff = "==0.4.7"
+coverage = "==7.5.3"
+djlint = "==1.34.1"
+pre-commit = "*"
```

Comment:
It's recommended to install a determined version. You can reinstall the libraries using `pipenv install library_name==version`
This will remove from here the "*" automatically.

Reply:
The same for the other libraries with "*"